### PR TITLE
Add Paris (France) Java User Group info's

### PR DESCRIPTION
**Pull request by kanedafromparis**

### CODE REVIEW

File: _jugs/ParisJug.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name:     "Paris Java Users Group"
```
Comment (sunix) [inline _jugs/ParisJug.md:2]: Users or User ?

**Comment by kanedafromparis** [comment]
looking at previous submit, both seems used 

- Vigo Java User**s** Group
- Warsaw Java User**s** Group
- Nairobi Java User Group

So...

**Comment by sunix** [comment]
In our website, it is 

> Bienvenue au Paris Java User Group ! 

So maybe we should keep it consistent. Anyway, not a big deal.

**Comment by kanedafromparis** [comment]
So we update our website ?

**Comment by sunix** [comment]
up to you:
- web site
- twitter
- etc.

or this PR ;)

File: _jugs/ParisJug.md (hunk 1 above)
Comment (icougil) [reply _jugs/ParisJug.md]: Well, if you have more than 1 user... 😉
I'm not English native, but I think the correct form should be in plural. 
For example, [here, in the JCP you can see the list of Java Users Groups](https://jcp.org/en/participation/JUG_list) where clearly we have 2 plurals, for users + groups

Comment (sunix) [reply _jugs/ParisJug.md]: yeah i think both are ok, but it's just to be consistent with our website/twitter/etc...

**Pull request merged**